ga4gh/datamodel/datasets.py

```diff
@@ -190,3 +237,69 @@ def _setMetadata(self):
                 except KeyError as err:
                     raise exceptions.MissingDatasetMetadataException(
                         metadataFileName, str(err))
+
+
+class AbstractOntology(object):
+    """
+    The base class of storing an ontology
+    At this moment an "Ontology" is just a map between names and IDs (e.g.
+    in Sequence Ontology we would have "SO:0001583 <-> missense_variant")
+    This is a tempotrary solution adn must be replaced by more comprehensive
+    ontology object.
+    """
+
+    def __init__(self):
+        self._nameIdMap = dict()
+        self._idNameMap = dict()
+
+    def add(self, id_, name):
+        self._nameIdMap[id_] = name
+        self._idNameMap[name] = id_
+
+    def getId(self, name):
+        return self._idNameMap[name]
+
+    def getName(self, id_):
+        return self._nameIdMap[id_]
+
+
+class FileSystemOntology(object):
```

Comment:
You may want to store the OBO file for the ontology as well for reasons of reproducibility, that's why I used a dir instead of a simple TXT file. When ontologies are properly implemented a single TXT file may not be enough.

OK, I'm changing FileSystemOntology -> FileSystemOntologies
and AbstractOntology -> FileSystemOntology
This should make more sense.

Added in a new module 'ontologies.py' as suggested.